_config.yml

```diff
@@ -11,7 +11,7 @@ permalink:      pretty
 # Server
 source:         "site"
 destination:    ./_gh_pages
-host:           "localhost"
+host:           "0.0.0.0"
```

Comment:
This was intentional.

Reply:
`localhost` won't work for someone using Docker.